frontend/package.json

```
@@ -1,5 +1,6 @@
 {
   "name": "@kiali/kiali-ui",
+  "version": "1.49.0",
```

Since 1.49.0 has already been released, this needs to be 1.50.0:
```suggestion
  "version": "1.50.0",
```